saas/mic-hex-eims/boot/src/main/resources/application-cloud.properties

```diff
@@ -0,0 +1 @@
+discovery.enabled=true
```

Comment:
I am not sure this property even exists? 
Probably a typo. 
```suggestion
eureka.client.enabled=true
```

Reply:
@sokimaaa, I also was puzzled, but set this property, according to your answer in messenger). I will fix this.